sdk/storage/azure-storage-blob/azure/storage/blob/_generated/aio/operations/_blob_operations.py

```diff
@@ -205,7 +205,7 @@ async def download(
         _decompress = kwargs.pop("decompress", True)
```

Comment:
The crux of the issue here is that `decompress` gets popped here and only used below in L396 to the deserializer call, but we need to pass this information along the pipeline `run()` to allow `_aiohttp.py` to adjust decompression behavior accordingly